src/main/java/org/opentripplanner/framework/io/HttpUtils.java

```diff
@@ -76,12 +80,15 @@ public static List<Header> getHeaders(
       );
     }
     if (response.getStatusLine().getStatusCode() != 200) {
-      throw new RuntimeException(
-        "Resource " +
-        sanitizeUri(uri) +
-        " unavailable. HTTP error code " +
+      LOG.warn(
+        "Headers of Resource {}" +
+        " unavailable. HTTP error code {}" +
+        " file size can't be logged.",
```

Comment:
file size can't be logged --> file size, creation date and content type can't be logged
It is not only the file size that cannot be logged, but also the creation date and the content type.
File size and creation date are used only for information purpose, but content type is used to determine if the source is a zip file.
Without the content type, the file extension is used to check if the file is a zip file

Reply:
you're right. That's probably confusing. I deleted that part of the log.